examples/sklearnex/gbt_cls_model_create_from_catboost_batch.py

```diff
@@ -0,0 +1,89 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation
```

Comment:
Looks that those older names were not removed

Reply:
fixed